decompressed/base/lib/mount_modroot/05_transfer_basefiles

```diff
@@ -23,7 +23,7 @@ preserve_base_file() { # <source> <dest>
 }
 
 #use the www folder as check to know that we should populate the modoverlay (mtd3)
-[ ! -d $overlay_bank/www ] && mv -f $overlay_bank/* $upperdir/
+[ -d $overlay_bank/www ] && mv -f $overlay_bank/* $upperdir/
```

Comment:
Ok, wait, se nelle intenzioni il senso sarebbe:
`faccio il mv solo quando in /overlay/bank_2 c'è ancora www`
allora qualcosa non torna

Reply:
Ah no scusa stavo guardando prima del commit, ora è giusto

Reply:
quando viene fatto il mount della nuova partizione è normale che /overlay sia sporca...

Reply:
why?
quello è l'mv che deve ripulire mtd2 e spostare tutto in mtd3
il preserve successivo riporta dietro i file che abbiamo decisono devono essere in mtd2

Reply:
per me il discorso fila